odoo/addons/base/models/ir_actions_report.py

```diff
@@ -1182,3 +1183,35 @@ def get_valid_action_reports(self, model, record_ids):
             if records.filtered_domain(literal_eval(action.domain)):
                 valid_action_report_ids.append(action.id)
         return valid_action_report_ids
+
+    @api.model
+    def _prepare_local_attachments(self, attachments):
+        attachments_with_data = self.env['ir.attachment']
+        for attachment in attachments:
+            if not attachment._is_remote_source():
+                attachments_with_data |= attachment
+            elif (stream := attachment._to_http_stream()) and stream.url:
+                """
+                _to_http_stream -> cloud_storage attachments
+                url -> url attachments
+                """
+                try:
+                    data = requests.get(attachment.url, timeout=10)
+                    data.raise_for_status()
+                    attachment_request = data.content
+                    if not attachment_request:
+                        _log.warning("Attachment %s at with URL %s retrieved successfully, but no content was found.", attachment.id, attachment.url)
+                        continue
+                    attachments_with_data |= self.env['ir.attachment'].new({
+                        'db_datas': attachment_request,
+                        'name': attachment.name,
+                        'mimetype': attachment.mimetype,
+                        'res_model': attachment.res_model,
+                        'res_id': attachment.res_id
+                    })
+                except requests.exceptions.RequestException as e:
+                    _logger.error("Request for attachment %s with URL %s failed.", attachment.id, attachment.url)
+                    _logger.error(e)
+            else:
+                _logger.warning("Unexpected edge case — apparently it lives here now.")
```

Comment:
I think it should have the same log level as logger in `requests.exceptions.RequestException`